ray_on_golem/server/services/golem/golem.py

```diff
@@ -46,6 +59,45 @@
 DEFAULT_PROPOSAL_RESPONSE_TIMEOUT = timedelta(seconds=30)
 
 
+class NoMatchingPlatform(AllocationException):
+    ...
+
+
+# FIXME: Rework this into the golem-core's DefaultPaymentManager and Allocation on yagna 0.16+,
+#  as until then there is no api call available to get driver lists and golem-core is api-only
+class DeviceListAllocationPaymentManager(DefaultPaymentManager):
+    @trace_span(show_arguments=True, show_results=True)
+    async def _create_allocation(self, budget: Decimal, network: str, driver: str) -> Allocation:
+        output = json.loads(
+            # await run_subprocess_output(YAGNA_PATH, "payment", "driver", "list", "--json")  # yagna 0.15+
+            await run_subprocess_output(YAGNA_PATH, "payment", "drivers", "--json")  # yagna 0.13
```

Comment:
I believe we have settled for a version check and a version dependent call here, haven't we?

Reply:
No longer needed, as yagna 0.15 is publicly available.